presto-docs/src/main/sphinx/develop/spi-overview.rst

```diff
@@ -5,10 +5,11 @@ SPI Overview
 When you implement a new Presto plugin, you implement interfaces and
 override methods defined by the SPI.
 
-Plugins can provide connectors, which are the source of all data
-for queries in Presto. Even if your data source doesn't have underlying
-tables backing it, as long as you adapt your data source to the API
-expected by Presto, you can write queries against this data.
+Plugins can provide additional connectors, types, functions, and system access control (see
```

Comment:
We can rename the title of `Type System` to `Types`, which is more consistent with `Connectors` and `Functions`